algebird-test/src/test/scala/com/twitter/algebird/MomentsLaws.scala

```diff
@@ -1,23 +1,41 @@
 package com.twitter.algebird
 
 import org.scalatest._
+import com.twitter.algebird.BaseProperties._
+import com.twitter.algebird.scalacheck.arbitrary._
+
+class MomentsLaws extends CheckProperties {
+  val EPS = 1e-10
+
+  property("Moments Group laws") {
+    groupLawsEq[Moments] { (ml, mr) =>
+      (ml.m0 == mr.m0) &&
+        approxEq(EPS)(ml.m1, mr.m1) &&
+        approxEq(EPS)(ml.m2, mr.m2) &&
+        approxEq(EPS)(ml.m3, mr.m3) &&
+        approxEq(EPS)(ml.m4, mr.m4)
+    }
+  }
+}
 
-class MomentsGroupTest extends WordSpec with Matchers {
-
-  /**
-   * Given a list of doubles, create a Moments object to hold
-   * the list's central moments.
-   */
-  def getMoments(xs: List[Double]): Moments =
-    xs.foldLeft(MomentsGroup.zero) { (m, x) => MomentsGroup.plus(m, Moments(x)) }
+object MomentsSpec {
```

Comment:
nope, deleted, that was waste from a shared function I had then killed. good catch.